docs/getting_started.rst

```diff
@@ -242,8 +242,10 @@ If you need to make the development server available through the LAN, you need t
 
   # first build the assets
   yarn run build
+  # collects the static files into KOLIBRI_HOME/static allowing them to be served by cherrypy
+  kolibri manage collectstatic
```

Comment:
Since https://github.com/learningequality/kolibri/pull/7920 was merged `collectstatic` is no longer necessary, as static files are now always served without collectstatic needing to be run. Can remove this.

Reply:
Ohh, okay sir, I've removing it.